cfgov/wellbeing/jinja2/wellbeing/home.html

```diff
@@ -20,6 +20,8 @@
 
 {% block css -%}
     {{ super() }}
+    <link rel="stylesheet"
+          href="{{ static('apps/financial-well-being/css/main.css') }}">
```

Comment:
Oops this can be deleted? `main.css` doesn't appear to have any styles that `home.html` uses?

Reply:
Correct, this will be removed.